COVID19/src/screens/Locations.js

```diff
@@ -70,7 +70,9 @@ const Location = () => {
         />
       </View>
       <View>
-        <TouchableOpacity style={styles.buttonContainer}>
+        <TouchableOpacity
+          onPress={() => navigation.navigate('Camera')}
+          style={styles.buttonContainer}>
           <Icon size={60} color="#5A4DCC" name="pluscircleo" />
```

Comment:
You should use the colors from your constant file

Reply:
Resolved!